Modules/Phasing.lua

```diff
@@ -665,12 +665,16 @@ function Phasing.IsSpawnVisible(phase)
         return (not complete[27769])
     end
 
+    if phase == phases.RHEA_NEW_KARGATH then
+        return complete[27769] and (questLog[27889] and questLog[27889].isComplete == 1) and ((not complete[27897]) and ((not questLog[27897]) or (questLog[27897] and questLog[27897].isComplete == 0))) or false
```

Comment:
Both of these quests don't have any objective:
1. `27889`
2. `27897`

That means they are always complete if they are in the quest log and `isComplete` is always true.

So the check can be simplified to:

```suggestion
        return complete[27769] and questLog[27889] and (not complete[27897]) and (not questLog[27897]) or false
```